pkg/cmd/server/origin/controller/quota.go

```diff
@@ -5,39 +5,44 @@ import (
 	"time"
 
 	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
-	quotacontroller "github.com/openshift/origin/pkg/quota/controller"
 	"github.com/openshift/origin/pkg/quota/controller/clusterquotamapping"
 	"github.com/openshift/origin/pkg/quota/controller/clusterquotareconciliation"
+	"github.com/openshift/origin/pkg/quota/image"
 	"k8s.io/kubernetes/pkg/controller"
 	kresourcequota "k8s.io/kubernetes/pkg/controller/resourcequota"
-
-	"github.com/openshift/origin/pkg/quota"
+	"k8s.io/kubernetes/pkg/quota/generic"
+	quotainstall "k8s.io/kubernetes/pkg/quota/install"
 )
 
 func RunResourceQuotaManager(ctx ControllerContext) (bool, error) {
 	concurrentResourceQuotaSyncs := int(ctx.OpenshiftControllerOptions.ResourceQuotaOptions.ConcurrentSyncs)
 	resourceQuotaSyncPeriod := ctx.OpenshiftControllerOptions.ResourceQuotaOptions.SyncPeriod.Duration
 	replenishmentSyncPeriodFunc := calculateResyncPeriod(ctx.OpenshiftControllerOptions.ResourceQuotaOptions.MinResyncPeriod.Duration)
 	saName := "resourcequota-controller"
+	listerFuncForResource := generic.ListerFuncForResourceFunc(ctx.GenericInformerFunc)
+	quotaConfiguration := quotainstall.NewQuotaConfigurationForControllers(listerFuncForResource)
```

Comment:
@derekwaynecarr guess this is enabling the generic quota count and fixes the bugzilla

Reply:
yes